cmd2/cmd2.py

```diff
@@ -1717,7 +1717,7 @@ def parseline(self, line: str) -> Tuple[str, str, str]:
         :return: tuple containing (command, args, line)
         """
         statement = self.statement_parser.parse_command_only(line)
-        return statement.command, statement.args, statement.command_and_args
+        return statement.command, statement, statement.command_and_args
```

Comment:
If I'm reading this correctly, the type hint for the return type of this function should change like so:
```Python3
def parseline(self, line: str) -> Tuple[str, Statement, str]:
```
Basically the middle item in the tuple should change from ``str`` to ``Statement``.

Reply:
This is an interesting question to consider. **parseline** is an override of a standard library `cmd` method. The original method returns a **str**. **Statement** is a subclass of **str**. Should:

- we return a **Statement** object or a **str**?
- If we decide to return a **Statement**, should the type hint match the method we are overriding, or the object we are returning?